app/dashboard/utils.py

```diff
@@ -385,6 +385,45 @@ def getBountyContract(network):
     getBountyContract = web3.eth.contract(standardbounties_addr, abi=bounty_abi)
     return getBountyContract
 
+def get_poap_contract_addresss(network):
+    if network == 'mainnet':
+        return to_checksum_address('0x22C1f6050E56d2876009903609a2cC3fEf83B415')
+    elif network == 'ropsten':
+        return to_checksum_address('0x50C5CA3e7f5566dA3Aa64eC687D283fdBEC2A2F2')
```

Comment:
Do you know if theres a rinkeby POAP contract? Typically we test using rinkeby instead of ropsten

Reply:
I couldn't find any POAP contract on ropsten as PAOP readme only mentioned rinkeby testnet [POAP readme](https://github.com/poapxyz/poap#deployed-contracts)

Reply:
ah bummer!